tasks/main.yml

```diff
@@ -1 +1,40 @@
 # role tasks
+
+- block:
+  - name: install NFS server packages
+    package:
+      name: "{{ item }}"
```

Comment:
Does `squash_actions` not take care of this?

https://docs.ansible.com/ansible/2.4/intro_configuration.html#squash-actions

Reply:
It might, but I think being explicit about passing the list directly to the task is more clear in reflecting the intent. The question of whether or not you *really* meant to `with_items/loop` the task or not would never come up.

That said, apparently `squash_actions` has been around since 2.0. I don't know how long it's defaulted to squashing the actions of the packaging plugins, but I've personally never seen it work even when explicitly looking for it, as recently as in ansible 2.3.

Reply:
I have not had success with squash actions working properly. Also, that seems to be a hack built in to work around people writing things dumb or for using ancient playbooks from before modules supported that. I'd say stick with passing the list in, rather than making the user optimize that with a questionably working setting. It's in our standards to pass the list rather than items at a time.